charts/grafana/templates/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.podDisruptionBudget }}
```

Comment:
Could you move this to templates/_helpers.tpl and define like this?

```
{{/*
Return the appropriate apiVersion for ingress.
*/}}
{{- define "grafana.psp.apiVersion" -}}
  {{- if and (.Capabilities.APIVersions.Has "policy/v1") (semverCompare ">= 1.21-1" .Capabilities.KubeVersion.Version) -}}
    {{- print "policy/v1" -}}
  {{- else -}}
    {{- print "policy/v1beta1 -}}
  {{- end -}}
{{- end -}}
```
and use as:
kind: {{ template "grafana.psp.apiVersion" . }}